From 9076e7134c6a90b8951deb21089e2108330e0e06 Mon Sep 17 00:00:00 2001
From: Harish Krupo <harish.krupo.kps@intel.com>
Date: Fri, 13 Oct 2017 15:06:33 +0530
Subject: [PATCH 14/15] libsync: Set num_fences in info

Set the num fences to the info variable in order to
retrieve the timestamp for all the fences

Change-Id: I46230d737fee2387552fb206191d7f2f54ee21ad
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-50575
Signed-off-by: Harish Krupo <harish.krupo.kps@intel.com>
---
 libsync/sync.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libsync/sync.c b/libsync/sync.c
index 0950082..8525b7d 100644
--- a/libsync/sync.c
+++ b/libsync/sync.c
@@ -220,6 +220,7 @@ static struct sync_file_info *modern_sync_file_info(int fd)
 
     info->num_fences = local_info.num_fences;
     info->sync_fence_info = (__u64)(uintptr_t)(info + 1);
+    info->num_fences = local_info.num_fences;
 
     err = ioctl(fd, SYNC_IOC_FILE_INFO, info);
     if (err < 0) {
-- 
1.9.1

